backend/src/data/repositories/interview/interview.repository.ts

```diff
@@ -31,9 +31,14 @@ class Interview {
     const interview = await this.#InterviewModel
       .query()
       .select()
-      .withGraphJoined('courseCategory')
-      .withGraphJoined('interviewee')
-      .withGraphJoined('interviewer')
+      .withGraphFetched('courseCategory')
+      .withGraphFetched('interviewee(withoutPassword).[userDetails]')
+      .withGraphFetched('interviewer(withoutPassword).[userDetails]')
```

Comment:
```suggestion
      .withGraphJoined('courseCategory')
      .withGraphJoined('interviewee(withoutPassword).[userDetails]')
      .withGraphJoined('interviewer(withoutPassword).[userDetails]')
```
Just tried with `withGraphFetched`, doesn't work as expected. Attaching screenshot of the result
![image](https://user-images.githubusercontent.com/61088675/187016234-cc3f4162-92cc-449c-8d6f-98fbf2714805.png)
Should be the same for other methods

Reply:
@fshabanov weird, I tested it locally, and withGraphFetched worked fine